pontoon/teams/templates/teams/widgets/team_list.html

```diff
@@ -48,10 +48,23 @@ <h4>
   </tr>
 {% endmacro %}
 
-{% macro footer(request=False) %}
+{% macro footer(request=False, form= None) %}
     </tbody>
   </table>
   {% if request %}
-    <button id="request-locales">Request new locales</button>
+    <form id="request-locale-form">
+      <div class="labels">
+        {{ form.name.label_tag() }}
+        {{ form.code.label_tag() }}
+      </div>
+      <div class="fields clearfix">
+        {{ form.name }}
+        {{ form.code }}
+      </div>
+      <div id="request-locales-note">
+        <p>The requested locale won’t be enabled automatically: an email will be sent to Pontoon admins and team managers to evaluate the request. We’ll try to respond as soon as possible.</p>
```

Comment:
There are no team managers to send the email to, so you should remove "and team managers" from the message.